crates/compiler/gen_dev/src/lib.rs

```diff
@@ -361,7 +337,8 @@ trait Backend<'a> {
         if ident_string.contains("#help") {
             format!("{}_{}_1", module_string, ident_string)
         } else {
-            format!("{}_{}_{}", module_string, ident_string, state.finish())
+            // Instead of wasting time hashing or looking at arguments, just use the InLayout index directly here.
+            format!("{}_{}_{}", module_string, ident_string, result.index())
```

Comment:
@ayazhafiz this is not stable between runs right. so this might cause caching issues in the future.